src/main/resources/userEmailFragment-sessionAdditionalContactInformationFragment.html

```diff
@@ -0,0 +1,9 @@
+<p>
+  <ul>
+    <li><strong>Requests for deadline extensions, problems regarding team/student data</strong> (e.g. wrong team, misspelled name),
+    and <strong>other non-technical queries</strong> about the feedback session :
+    <br>Contact the instructors of the course: ${coOwnersEmails}</li>
+    <li><strong>Technical help regarding TEAMMATES</strong> (e.g. submission link is not working) :
```

Comment:
To make it consistent, there should be no space before `:`. Similarly to the entry two before this.

Reply:
Okay